src/services/activeUsers.js

```diff
@@ -0,0 +1,124 @@
+import { Query } from 'pg';
+import through2 from 'through2';
+
+import {
+  sequelize,
+} from '../models';
+
+const arrayFields = [
+  'Roles',
+  'READ_WRITE_REPORTS regions',
+  'READ_REPORTS regions',
+  'APPROVE_REPORTS regions',
+  'UNLOCK_APPROVED_REPORTS regions',
+  'Flags',
+];
+
+/**
+ * Transform function to convert the data to the right format.
+ *
+ * @param {*} user - a user row
+ */
+function convertToCSV(user) {
+  // Surround array fields with double quotes
+  const mappedUser = {};
+  Object.keys(user)
+    .forEach((k) => { mappedUser[k] = (arrayFields.includes(k) && user[k] ? `"${user[k]}"` : user[k]); });
+
+  const result = Object.values(mappedUser).toString();
+
+  return `${result}\n`;
+}
+
+/**
+ * Transform function to convert the data to the right format.
+ *
+ * @param {*} sql - raw sql
```

Comment:
minor, no need to prioritize it, but its a freebie to add the types to the jsdoc
```js
/*
 * @param {string} sql - raw sql
 */
```
etc

Reply:
at least the simpler ones, I wouldn't personally bother with the objects, like the user one in convertToCSV